tests/misc.js

```diff
@@ -244,3 +244,30 @@ test('expose stats.peers', function (t) {
     })
   })
 })
+
+test('expose .downloaded', function (t) {
+  rimraf.sync(path.join(shareFolder, '.dat'))
+  var downFolder = path.join(os.tmpdir(), 'dat-' + Math.random().toString(16).slice(2))
+  fs.mkdirSync(downFolder)
+
+  var shareDat = Dat({ dir: shareFolder, snapshot: true, db: memdb() })
```

Comment:
We should add a test with snapshot: false, since that is the more complex situation. We need to think about how this will work when a download finished, source is updated, and download starts again.